x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/FieldExtractorTestCase.java

```diff
@@ -1107,6 +1108,445 @@ public void testTypeConflictInObject() throws IOException {
         );
     }
 
+    /**
+     * https://github.com/elastic/elasticsearch/issues/117054
+     * One index with (Responses.process is nested; process.parent.command_line is supported):
+     * <pre>
```

Comment:
Nit: I'd leave the mapping out of the javadoc, since its already legible in the code.